analyzers/src/SonarAnalyzer.CSharp/Rules/RedundantInheritanceListCodeFixProvider.cs

```diff
@@ -67,12 +58,19 @@ protected override Task RegisterCodeFixesAsync(SyntaxNode root, CodeFixContext c
             return TaskHelper.CompletedTask;
         }
 
+        internal static bool HasLineEnding(SyntaxNode node) =>
```

Comment:
internal should go before protected

Reply:
Indeed, yet in this case because it is static it is OK to leave it like this. According to the [coding style](https://github.com/SonarSource/sonar-dotnet/blob/master/docs/coding-style.md) : Static methods are preferred to be after instance methods.